src/osd/OSDMap.cc

```diff
@@ -57,6 +57,9 @@ using ceph::decode;
 using ceph::encode;
 using ceph::Formatter;
 
+#ifndef WITH_SEASTAR
+  #include "common/Cond.h"
+#endif
```

Comment:
OSDMap.h has a WITH_SEASTAR ifdef for the smart pointer type.  Other than that, I really don't want OSDMap to have compile-time differences for WITH_SEASTAR.